files/image_config/monit/mgmt_oper_status

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+"""
+"""
+
+import docker
+import sys
+import subprocess
+import syslog
+
+from sonic_py_common import multi_asic, device_info
+from swsscommon.swsscommon import SonicV2Connector
+
+
+def main():
+    db = SonicV2Connector(use_unix_socket_path=True)
+    db.connect('CONFIG_DB')
+    db.connect('STATE_DB')
+    mgmt_ports_keys = db.keys(db.CONFIG_DB, 'MGMT_PORT|*' )
+    if not mgmt_ports_keys:
+        syslog.syslog(syslog.LOG_DEBUG, 'No management interface found')
+    else:
+        try:
+            mgmt_ports = [key.split('MGMT_PORT|')[-1] for key in mgmt_ports_keys]
+            for port in mgmt_ports:
+                state_db_key = "MGMT_PORT_TABLE|{}".format(port)
+                # Reset status of mgmt port before updating with latest status
+                db.set(db.STATE_DB, state_db_key, 'oper_status', 'unknown')
```

Comment:
Can we do a read first and only when there is a change of oper state, the log there is a change whether it be from UP to Down or Down to UP transition.
Let's not reset it to unknown as we want to avoid making any changes unless there is a need to do so (state change detected).  For the state cahnge from UP to Down, let.s make it as a warning. For state change from down to up let's make it as a INFO level syslog.

Reply:
modified as suggested